### ISSUE TITLE
error: The method 'FormBuilderDateTimePicker' isn't defined for the type

### ISSUE THREAD

**Issue by niklas-b** [danvick/flutter_form_builder#775]
I just upgraded to flutter_form_builder: ^6.0.0-nullsafety.1 and am getting the following error:

```
The method 'FormBuilderDateTimePicker' isn't defined for the type ...

Try correcting the name to the name of an existing method, or defining a method named 'FormBuilderDateTimePicker'.
```

The documentation for 6.0.0 mentions that FormBuildDateTimePicker is supported... Is it not available in 6.0.0+ ? Or is there some other error and if so, anything I can do on my end to make it work?

**Comment by angwandi** [comment]
Same issue here

**Comment by Clid3** [comment]
same issue here

**Comment by NiranjanShah** [comment]
Same issue here, appreciate your response at the earliest please.

BTW, form_builder_date_time_picker.dart is fully commented.......

**Comment by jaxnz** [comment]
Looks fixed with with latest release ef2892d9d70423677484d64579f195b2ccbdb28c
